doc/api_assets/style.css

```
@@ -328,6 +328,14 @@ li.picker-header a span {
   background-color: var(--blue1);
 }
 
+.api_warning {
+  border: 5px solid var(--red2);
```

```suggestion
  border: 5px solid var(--red2);
  border-radius: 5px;
```

Adding a border raduis can soften the "box".